### DISCUSSION TITLE
Idea for this mod: Photographer Villager Quests

### DISCUSSION THREAD

**Discussion by BlockyUniverse** [mortuusars/Exposure#67] [Ideas]
In short, the idea is to have a villager (and related villager house) that is dedicated to photography, and while they would sell equipment needed for the camera, they can also make "requests" for certain photos in exchange for rewards.

**Comment by mortuusars** [comment]
This is a cool idea, but I think it's better to implement it in a modpack rather than include it in a mod itself.
